### PR TITLE
Make demographics editable

### PR DESCRIPTION
**Pull request by karlhigley**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -56,10 +56,6 @@ def email_redirect(path):
         account_repo.store_login(data)
         conn.commit()
 
-    # redirect for deprecated endpoint.
```
[the remark this answers is not in the document]
Comment (karlhigley) [reply app.py:59]: I suspect it has something to do with a weird merge resolution when merging `main` into a branch, but I couldn't tell you exactly what

File: app.py [hunk 2]
```diff
@@ -222,11 +227,12 @@ def topics():
         ("Not at all interested", 1),
     ]
 
-    def get_topic_preferences(account_id):  # for geting user topic preference
+    def get_topic_preferences(account_id): #for geting user topic preference
         with DB_ENGINE.connect() as conn:
             repo = DbAccountInterestRepository(conn)
             preferences = repo.fetch_topic_preferences(account_id)
         preferences_dict = {pref.entity_name: pref.preference for pref in preferences}
+        print(preferences_dict)
```
[the remark this answers is not in the document]
Comment (Sris-Ty) [reply app.py:235]: Sorry, I was just checking the information_dict there. I will remove it.

File: templates/onboarding_survey.html [hunk 3]
```diff
@@ -2,50 +2,82 @@
 {% block header %}
 <script>
```
[the remark this answers is not in the document]
Comment (Sris-Ty) [reply templates/onboarding_survey.html:3]: I guess I just tested it up to the point that it works for the "other" option and "not listed" option and with the combination of other checked races and email_clients and not checking the "other" and "not listed" options.

File: app.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (Sris-Ty) [reply app.py:59]: Yes, it was not an intentional one. I didn't even notice that I removed that one.

File: app.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (Sris-Ty) [reply app.py]: Removed the print statements

**Comment by ShahanM** [comment]
I merged the main branch to this branch since most of the breaking code from the main branch were mine.
Also, I fixed some minor validation issues which I either nuked during the merge or might have missed before.

**Comment by ShahanM** [comment]
Oh also, @Sris-Ty could you look at it and make sure I did not remove or make any undesired changes to your code.

**Review by ShahanM — APPROVED** [review]
Looks good to me. I made minor changes but primarily to facilitate the merging and conflict resolution.

**Pull request merged**